services/website/content/en/identities/wasif.md

```diff
@@ -0,0 +1,17 @@
+---
+template: IdentityDetailPage
+title: Wasif Hassan Baig
+description: SPO
+image: https://github.com/ruby-pool/ruby-web-app/blob/main/w.png?raw=true
+website: https://rubypools.com
+telegram: wbayg
+twitter: wbayg
+github: baig
+donationAddress: addr1q9wrjegg76eehpxjzvjc5yk8edft3rrzeghzpz0ukumf5v4420mxq8d5xx6qy8srmz4x7nfm99vuqwppxk2g3us39u7sa67av7
+verified: true
+badges:
+  - name: Fully ARMed
+    color: "#EF4444"
+  - name: ARMada CAP'N
+    color: "#846DCF"
```

Comment:
The badges property has to be removed.

Since you're not over on ARM yet, you're not eligible to the Fully ARMed badge. This badge will automatically be generated in the future.

The ARMada CAP'N badge is still under review based on contributions.